test/unit_tests/graphql/test_product.py

```diff
@@ -202,11 +207,18 @@ def test_product_has_previous_page(test_client, generic_product_1, generic_produ
     assert product_names == ["Product 2", "Product 3", "ProductSubListUnion", "ProductSubOne", "ProductSubTwo"]
 
 
-def test_products_filter_by_product_block(test_client, generic_product_1, generic_product_2, generic_product_3):
-    data = get_product_query(
-        filter_by=[{"field": "product_blocks", "value": "PB_1-PB_3"}],
-        sort_by=[{"field": "name", "order": "ASC"}],
-    )
+@pytest.mark.parametrize(
+    "query_args",
+    [
+        {"filter_by": [{"field": "product_blocks", "value": "PB_1-PB_3"}]},
+        {"query_string": "product_block:(PB_1|PB_3)"},
+        # {"query_string": f"productBlock:PB_1 | product_block:PB_3"}, Doesn't work yet :(
```

Comment:
Do we intend to fix that? :) If not let's just remove it. Otherwise you could mark this scenarios as an xfail (expected failure) so that it at least runs https://docs.pytest.org/en/6.2.x/skipping.html#skip-xfail-with-parametrize

Reply:
Thanks. Should have added a TODO. It was fixed